deployments/kubernetes/compute.dev.yaml

```diff
@@ -82,6 +82,10 @@ spec:
         env:
         - name: "TEMPORAL"
           value: "temporaltest-frontend.default.svc.cluster.local"
+        - name: RPS
+          value: "1000"
+        - name: CONCURRENCY
+          value: "1000"
```

Comment:
@phudtran Can you write down how to modify and apply these two parameters during the compute deployment and after compute deployed?

Reply:
Done 👍